source/default/_patterns/00-protons/legacy/js/utc-hover-images.js

```diff
@@ -0,0 +1,47 @@
+(function($, Drupal, drupalSettings) {
+    "use strict";
+   //Adds slick arrows 
+    Drupal.behaviors.slickcustomarrows = {
```

Comment:
slickarrows are for when it goes mobile i supposed.

Reply:
Oh sorry, that's a cut an paste error. Let me fix it.